renderer/components/ui/DeleteTooltop.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
Rename the name of the file from DeleteTooltop to DeleteTooltip. I guess it will be right